app/views/users/_admin_for_projects.html.haml

```diff
@@ -12,6 +12,6 @@
           = p.name
   %div.additional-options
     = hidden_field(:user, :can_add_teachers_to_cohorts, :value => false)
-    = check_box_tag "user[can_add_teachers_to_cohorts]", 1, policy(@user).add_teachers_to_cohorts?
-    = label_tag "user_can_add_teachers_to_cohorts" do
-      User can add teachers to cohorts of projects they administrate
+    = check_box_tag "user[can_add_teachers_to_cohorts]", 1, policy(@user).add_teachers_to_cohorts?, id: "user_can_add_teachers_to_cohorts_cb"

```

Comment:
Based on this documentation: `check_box_tag(name, value = "1", checked = false, options = {})` the checkbox will be 'checked' based on the value of the policy. I think that is what you wrote. 

That isn't how it should work. The value for whether it is checked should be based on value in the database for the @user record.  The `policy(@user)...` should probably not be used here. 

In this form we want to know if a admin can add this setting to another admin. We could add another policy method here. But I think it'd be fine just to check if current_user is a portal admin. We should check the controller that handles this setting to make sure non portal admins cannot set this value on another user.